src/gluonnlp/models/electra.py

```diff
@@ -96,6 +163,7 @@ def get_generator_cfg(model_config):
 FILE_STATS = load_checksum_stats(os.path.join(get_model_zoo_checksum_dir(), 'electra.txt'))
 
 
+# TODO(sxjscience) Use BertTransformer
```

Comment:
@ZheyuYe I think we may try to directly reuse BertTransformer. Do you think it's possible?

Reply:
Yeah we could do that, encoders in bert and electra are almost same.

Reply:
OK, so let me keep the TODO and we can revise later.